starknet_devnet/blueprints/feeder_gateway.py

```diff
@@ -45,6 +45,21 @@ def _get_block_object(block_hash: str, block_number: int):
 
     return block
 
+def _get_block_transaction_traces(block):
+    traces = []
+    if block.transaction_receipts:
```

Comment:
Why is this check now necessary?

Reply:
It is not related to these changes, someone reported or I notices a use-case bug.